solr_conf/conf/solrconfig.xml

```diff
@@ -158,16 +169,27 @@
     </lst>
   </requestHandler>
 
+
   <requestDispatcher handleSelect="true" >
     <requestParsers enableRemoteStreaming="true" multipartUploadLimitInKB="2048" />
   </requestDispatcher>
 
   <requestHandler name="/analysis/field" startup="lazy" class="solr.FieldAnalysisRequestHandler" />
   <requestHandler name="/update" class="solr.UpdateRequestHandler"  />
+
   <updateHandler class="solr.DirectUpdateHandler2">
     <updateLog>
       <str name="dir">${solr.data.dir:}</str>
     </updateLog>
+    <autoCommit>
```

Comment:
Auto commit affects two things:
1) durability which is determined by hard commit
2) visibility which is determined by soft commit

For HydraNorth, my understanding is that it is a not a heavy indexing
application. So whenever a file is indexed, you might want it to be visible
to the users right away. So,
we might need to set OpenSearcher as true for soft commit (the side effects
are caches might be invalidated).

Please let me know what you care most and we can set up auto commit in a
way that suits your need.

On Fri, Sep 4, 2015 at 10:25 AM, pgwillia <notifications@github.com> wrote:

> In solr_conf/conf/solrconfig.xml
> <https://github.com/ualbertalib/HydraNorth/pull/559#discussion_r38768805>:
>
> >    <updateHandler class="solr.DirectUpdateHandler2">
> >      <updateLog>
> >        <str name="dir">${solr.data.dir:}</str>
> >      </updateLog>
> > +    <autoCommit>
>
> Just curious how the autocommit values were chosen?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/pull/559/files#r38768805>.
>